src/Uno.UI/ts/WindowManager.ts

```diff
@@ -1135,18 +1135,6 @@ namespace Uno.UI {
 			}
 		}
 
-		/**
-			* Destroy a html element.
-			*
-			* The element won't be available anymore. Usually indicate the managed
-			* version has been scavenged by the GC.
-			*/
```

Comment:
nit: Consider moving the comment to either `destroyViewNativeFast` or `destroyViewInternal`.